wgpu-core/src/command/bundle.rs

```diff
@@ -1238,6 +1298,8 @@ impl<A: HalApi> State<A> {
 /// Error encountered when finishing recording a render bundle.
 #[derive(Clone, Debug, Error)]
 pub(super) enum RenderBundleErrorInner {
+    #[error("resource is not valid to use with this render bundle")]
```

Comment:
```suggestion
    #[error("resource is not valid to use with this render bundle because the resource and the bundle come from different devices")]
```

In an ideal world we would also say which resource, and the identities of both devices, but I'm not sure we have all the infrastructure for that right now